pyproject.toml

```diff
@@ -1,14 +1,24 @@
 [tool.poetry]
-name = "pyap"
-version = "0.3.1"
-description = "Pyap is an MIT Licensed text processing library, written in Python, for detecting and parsing addresses. Currently it supports USA, Canadian and British addresses."
-authors = ["Vladimir Goncharov <vladimarius@gmail.com>"]
-documentation = "https://github.com/vladimarius/pyap"
+name = "argyle-pyap"
+version = "0.0.0"
+description = "Argyle's Pyap is a fork of Pyap, a MIT Licensed text processing library, written in Python, for detecting and parsing addresses. Currently it supports USA, Canadian and British addresses."
```

Comment:
maybe just adding a small comment about why this fork. Like `is a fork of Pyap` ... `with a few improvements and supporting a lot more addresses formats`